### PR TITLE
AMS David Zlobinskyy - Finalize semantic HTML structure and CSS styling of NPM website

### PR DESCRIPTION
**Pull request by zlobizlobi**

### CODE REVIEW

**Comment by IronhackAMS** [comment]
Hi there Zlobinskyy! Awesome nickname;) Me like!

Some general remarks:
- Indenting: try to indent sections and div’s within the body element. This makes it more readible.
- You are thinking in grids with the use of columns, good idea! That you already understand this notion is very important for the coming lessons. What you could improve: The “build amazing things block” is not yet in the middle, maybe you could solve this by adding extra columns on left and right so that the total number of columns in the parent `<div class=“bottom-header”>` container is a total sum of 12 (= total grid for full width of screen). Other solution and less neat, you could solve this by expressing width in % instead of px:-) 
- Try not to use fixed heights/widths when possible, especially not parent containers (exception can be made with background img, depending on the context). This can interfere with different elements in and out of the section container when you adjust screen size. Always try to size containers relative to their content and let the children component decide the size of its parent container. That way the components adjust nicely to the size of the screen. So try to remove the fixed heights and widths of these and see if you can come up with a different solution;) To see what I mean, open developer tools and adjust the screen size.
- You used article tags for children components. It’s convention to use this tag only for parent container’s. It’s like a body tag but then for writing articles:)

In conclusion: apart from some minor improvements, you are on the right track my friend! Well done!

If you have any questions, just slack us!:)

Kido
